client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDatabaseMetaData.java

```diff
@@ -1585,6 +1612,16 @@ private void assertMetadataCalls(
         };
     }
 
+    private int countRows(ResultSet resultSet)
+            throws Exception
```

Comment:
Nit: `SQLException`

(we often use `Exception` for `@Test` methods, which I think is fine, but I prefer utility methods to use correct exceptions when possible)

Reply:
We should use `throws Exception` in test code, including utilities, since this avoids redundant changes later.